listenbrainz/webserver/static/js/src/user-feed/UserFeed.tsx

```diff
@@ -530,7 +531,10 @@ document.addEventListener("DOMContentLoaded", () => {
   const domContainer = document.querySelector("#react-container");
   const propsElement = document.getElementById("react-props");
   const reactProps = JSON.parse(propsElement!.innerHTML);
-  const { api_url, current_user, spotify, events } = reactProps;
+  const { api_url, current_user, spotify, events, sentry_dsn } = reactProps;
```

Comment:
do we have types configured for these reactProps? @MonkeyDo

Reply:
Negative.
I guess it comes right back to our experiment with exporting TS types from Pydantic models…